File: backend/src/main/java/com/voltpay/backend/repository/ContactsBackupRepository.java
package com.voltpay.backend.repository;

import com.voltpay.backend.model.ContactBackup;
import org.springframework.data.mongodb.repository.MongoRepository;
import java.util.Optional;

public interface ContactsBackupRepository extends MongoRepository<ContactBackup, String> {
    Optional<ContactBackup> findByPhoneNumber(String phoneNumber);
}

File: backend/src/main/kotlin/com/voltpay/backend/repository/ContactsBackupRepository.kt
package com.voltpay.backend.repository

import com.voltpay.backend.model.ContactBackup
import org.springframework.data.mongodb.repository.MongoRepository
import java.util.Optional

interface ContactsBackupRepository : MongoRepository<ContactBackup, String> {
    fun findByPhoneNumber(phoneNumber: String): Optional<ContactBackup>
}
